elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -490,9 +551,12 @@ def grade(element_html, data):
         if final_score < 1:
             if feedback_type == 'none':
                 feedback = ''
-            elif feedback_type == 'first-wrong':
+            elif feedback_type.startswith('first-wrong'):
                 if first_wrong == -1:
                     feedback = FIRST_WRONG_FEEDBACK['incomplete']
+                elif feedback_type.endswith('-verbose') and student_answer[first_wrong]['inner_html'] in distractor_feedback:
+                    text = student_answer[first_wrong]['inner_html']
+                    feedback += FIRST_WRONG_FEEDBACK['distractor-feedback'].format(str(first_wrong + 1), text, distractor_feedback[text] or '')
```

Comment:
we're not doing it like this anymore, right? we are just going to have `feedback='first-wrong'` and then `distractor-info` be separate things?

Reply:
Yeah, I still need to separate them. Should have that updated in the next round of changes.